workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -96,6 +107,53 @@ def dump_stage(self, tempdir):
             raise NonRecoverableError('Error during dumping Stage data, '
                                       'exception: {0}'.format(ex))
 
+    def _get_admin_user_restore_command(self, dump_path):
+        with open(dump_path) as dump_handle:
+            dump = dump_handle.readlines()
+
+        copy_line = None
+        admin_user = None
+        for line in dump:
+            if line.startswith('COPY users ('):
```

Comment:
i wish there was another way of doing this - perhaps also add a new file to store the admin user